### ISSUE TITLE
Planned release compatible with Rails 5.2?

### ISSUE THREAD

**Issue by Obi-TOB** [globalize/globalize#696]
Hi,

I'm using gems that depend on globalize under Rails 5.2.

As I understand gemspecs do not allow to add dependencies based on branches, only based on releases, to avoid (additional) dependecy hell.

I.e. I need to manually workaround with bundler dependencies to get the gems up and running under Rails 5.2. I'm sure I'm not the only one having to deal with this.

I understand that the master branch is already Rails 5.2 "ready"? Is there any release planned?

**Comment by kevinhq** [comment]
Me too. When upgrading from Rails 5.1.6 to 5.2.1 , I got dependency error. Waiting for another release which compatible with Rails 5.2.1

**Comment by wbotelhos** [comment]
Hi @Obi-TOB and @kevinhq ,

For now, you can use the master. It worked for me:

```
gem "globalize", github: "globalize/globalize", ref: "HEAD"
```

**Comment by CaioBianchi** [comment]
Pulling a gem from the master branch is not really a future-proof solution.

**Comment by mzagaja** [comment]
I am currently blocked by this and would appreciate a new release since as far as I can tell I cannot point to a GitHub ref in a .gemspec file for refinery-blog.

**Comment by kevinhq** [comment]
@wbotelhos - We ended up removing this gem and work on our own solution. Yes, we did use that solution and it indeed works, but it's not really a future-proof solution as @CaioBianchi said.

**Comment by wbotelhos** [comment]
Now I'm using Mobility, but with some problems:

- Order was not work, but now fixed: https://github.com/shioyama/mobility/pull/261
- Now conditionals with more than one level of deep does not works: https://github.com/shioyama/mobility/issues/266

Will your solution be open @kevinhq  ?

**Comment by parndt** [comment]
Sorry all, I have not got much time available at all for Globalize lately. Is the master branch working for everyone who has tried it? If so, we can probably just release it as 5.2.0.

I've just merged a bunch of bug fixes so if anyone can check against their applications, that would really help.

Also recommend checking out @shioyama's Mobility library if it works for you.  Here's where I moved my major use case for Globalize, Refinery CMS, to Mobility: https://github.com/refinery/refinerycms/pull/3360 😄
